src/main/scala/wdlTools/util/Util.scala

```diff
@@ -193,36 +195,44 @@ object Util {
     *  trimmed, as well as `w` whitespace characters trimmed from the
     *  beginning of each line.
     *  @example
-    *    val s = "   \n  hello\n   goodbye  "
-    *    stripLeadingWhitespace(s) => (1, 2, "hello\n goodbye")
+    *    val s = "   \n  hello\n   goodbye\n "
+    *    stripLeadingWhitespace(s, false) => (1, 1, "hello\n  goodbye\n")
+    *     stripLeadingWhitespace(s, true) => (1, 2, "hello\n goodbye")
     */
-  def stripLeadingWhitespace(s: String): (Int, Int, String) = {
-    val lines = s.split(System.lineSeparator())
-    val wsRegex = "^\\s*$".r
-    val nonWsRegex = "^(\\s.*)(.*)$".r
-    val (lineOffset, content) = lines.foldLeft((0, Vector.empty[(Int, String)])) {
+  def stripLeadingWhitespace(s: String,
+                             ignoreEmptyLast: Boolean = false,
+                             lineSep: String = System.lineSeparator()): (Int, Int, String) = {
+    val lines = s.split("\r\n?|\n")
+    val wsRegex = "^([ \t]*)$".r
+    val nonWsRegex = "^([ \t]*)(.+)$".r
+    val (lineOffset, content) = lines.foldLeft((0, Vector.empty[(String, String)])) {
       case (l, wsRegex(_)) if l._2.isEmpty => (l._1 + 1, l._2)
```

Comment:
Could you please replace the `_1, _2` shorthand with meaningful variable names? It is harder to understand this way.